inc/gpio.h

```diff
@@ -65,10 +68,10 @@ typedef enum {
 // Address is separated to save memory and make the interface cleaner
 // Struct is 4 bytes and will fit in one memory register.
```

Comment:
I would remove the comment on size - it's not something you should rely on (and the single register thing isn't true for the msp430)